app/models/icon.rb

```diff
@@ -22,7 +22,7 @@ class Icon < ApplicationRecord
   before_validation :use_icon_host
   before_save :use_https
   before_update :delete_from_s3
```

Comment:
Maybe this should actually be an `after_update_commit`? 🤔 

Could be a future PR though, imo